<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDeliveryWindow</class>
 <widget class="QDialog" name="MyDeliveryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>191</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type Delivery</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adress</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>52</y>
     <width>49</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>16</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>- - - - DELETE - - - -</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Titl_delete">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>72</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>174</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>112</y>
     <width>81</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Type Delivery:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Type">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>132</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ukrposhta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Novaposhta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meest</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>add new Delivery</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>247</y>
     <width>570</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>close and save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>215</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order by:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Order">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Title</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Adress</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_Add_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
